meta-mender-core/recipes-bsp/u-boot/patches/0001-Add-missing-header-which-fails-on-recent-GCC.patch

```diff
@@ -1,23 +0,0 @@
-From 2a84a94f31981c00efcdbe4a907bdffd9601caf4 Mon Sep 17 00:00:00 2001
```

Comment:
Good to have this removed, but this got mixed into the wrong commit. There's at least three different commits worth of changes here:

* The original rpi forward port, which is good.
* The removal of the "missing header" patch.
* The removal of the experimental boot counter patch.

Especially the middle one is subject to possible revert later if we discover problems, so I think it's important to keep these changes separate.